ci/build_container/build_container.sh

```diff
@@ -32,10 +26,6 @@ pip install virtualenv
 export GOPATH=/usr/lib/go
 go get github.com/bazelbuild/buildifier/buildifier
 
-# GCC 4.9 for everything.
-export CC=gcc-4.9
-export CXX=g++-4.9
-
```

Comment:
It's probably still good to see `CC=gcc` etc. just to ensure that we don't accidentally get defaulted into clang one day by Bazel magic.